src/fragments/lib/auth/js/getting-started-steps-basic-auth-react-native.mdx

```diff
@@ -13,12 +13,12 @@ You will also need to install the pod dependencies for iOS:
 npx pod-install
 ```
 </Block>
-<Block name="Expo">
+<Block name="Expo CLI">
 
 Install the necessary dependencies by running the following command:
 
 ```sh
-npm install aws-amplify amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage core-js
+npm install aws-amplify amazon-cognito-identity-js amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage core-js
```

Comment:
I'd remove the duplicate installation here :) 

```suggestion
npm install aws-amplify amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage core-js
```

Reply:
Thanks for taking care of this @abdallahshaban557 and nice catch @david-mcafee!